test/unit/UnittestHTTP.c

```diff
@@ -1,8 +1,10 @@
-#include "CException.h"
 #include "Esp.h"
 #include "EspUnitTest.h"
 #include "HTTP.h"
+
+#include "CException.h"
```

Comment:
since there are many changes in this PR that reorder include statements: are you using a tool for this? If not, how will we keep this consistent? Why would we not reorder it again in the next commit? IE., if there's no tool, there should be rule.

Reply:
It is just done by clang-format by my IDE

Reply:
alright, sounds good.